tests/unit/QueryEditor.test.tsx

```diff
@@ -113,12 +113,12 @@ describe('QueryEditor RAW', () => {
     expect(onRunQuery).toHaveBeenCalledTimes(1);
   });
 
-  it.skip.each`
```

Comment:
typo in the commit message (can not comment on commit message?! )-:

Reply:
Could you tell me which message you mean? I didn't see one when proofreading just now.

Reply:
https://github.com/tribe29/grafana-checkmk-datasource/pull/126/commits/c636e87834db865a47feaccaf2e81e845a214d6f

> `it is and is`

feels like this was reworded and some words went missing/duplicated?